app/config/routing.yml

```diff
@@ -1,3 +1,10 @@
 app:
     # The main bundle is PrestaShopCoreBundle which will load other dependencies.
     resource: "@PrestaShopBundle/Resources/config/routing.yml"
+
+app_modules:
+    # Declare routing.yml file in modules/foo/config folder.
+    # v1: only YAML format is supported for now.
+    resource: .
+    type: module
+    prefix: /modules
```

Comment:
I fear this may conflict with routes we have already... but as I'm working on [better application routing](https://github.com/PrestaShop/PrestaShop/pull/9137), this won't be an issue in 1.7.5.

Reply:
In your comment, is `foo` the name of the module added automatically?

Reply:
no, but I should replace foo by "module-name"